client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -148,35 +152,74 @@ class TransferDomainPrecheck extends React.PureComponent {
 		const step = 1;
 		const isStepFinished = currentStep > step;
 
-		const heading = unlocked
-			? translate( 'Domain is unlocked.' )
-			: translate( 'Unlock the domain.' );
-		const message = unlocked
-			? translate( 'Your domain is unlocked at your current registrar.' )
-			: translate(
-					"Your domain is locked to prevent unauthorized transfers. You'll need to unlock " +
-						'it at your current domain provider before we can move it. {{a}}Here are instructions for unlocking it{{/a}}. ' +
-						'It might take a few minutes for any changes to take effect.',
-					{
-						components: {
-							a: (
-								<a
-									href={ support.INCOMING_DOMAIN_TRANSFER_PREPARE_UNLOCK }
-									rel="noopener noreferrer"
-									target="_blank"
-								/>
-							),
-						},
-					}
-				);
+		let heading = translate( 'Lock status unavailable.' );
+		if ( true === unlocked ) {
+			heading = translate( 'Domain is unlocked.' );
+		} else if ( false === unlocked ) {
+			heading = translate( 'Unlock the domain.' );
+		}
+
+		let message = translate(
+			"{{notice}}We couldn't get the lock status of your domain from your current registrar.{{/notice}} If you're sure your " +
+				"domain is unlocked then, you can continue to the next step. If it's not unlocked, then the transfer won't work. " +
+				'{{a}}Here are instructions to make sure your domain is unlocked.{{/a}}',
+			{
+				components: {
+					notice: <Notice showDismiss={ false } status="is-warning" />,
+					br: <br />,
+					a: (
+						<a
+							href={ support.INCOMING_DOMAIN_TRANSFER_PREPARE_UNLOCK }
+							rel="noopener noreferrer"
+							target="_blank"
+						/>
+					),
+				},
+			}
+		);
+		if ( true === unlocked ) {
+			message = translate( 'Your domain is unlocked at your current registrar.' );
+		} else if ( false === unlocked ) {
+			message = translate(
+				"Your domain is locked to prevent unauthorized transfers. You'll need to unlock " +
+					'it at your current domain provider before we can move it. {{a}}Here are instructions for unlocking it{{/a}}. ' +
+					'It might take a few minutes for any changes to take effect.',
+				{
+					components: {
+						a: (
+							<a
+								href={ support.INCOMING_DOMAIN_TRANSFER_PREPARE_UNLOCK }
+								rel="noopener noreferrer"
+								target="_blank"
+							/>
+						),
+					},
+				}
+			);
+		}
+
 		const buttonText = translate( "I've unlocked my domain" );
 
-		let lockStatusClasses = unlocked
-			? 'transfer-domain-step__lock-status transfer-domain-step__unlocked'
-			: 'transfer-domain-step__lock-status transfer-domain-step__locked';
+		let lockStatusClasses = 'transfer-domain-step__lock-status transfer-domain-step__unlocked';
+		if ( false === unlocked ) {
+			lockStatusClasses = 'transfer-domain-step__lock-status transfer-domain-step__locked';
+		} else if ( null === unlocked ) {
```

Comment:
For consistency sake, let's keep it as above. `null` default, `true/false` in the conditions. It's a bit confusing seeing it changed, so I expected special behavior. :)

Reply:
Ah...yeah. I meant to normalize this.